include/fiction/algorithms/physical_design/automatic_exhaustive_gate_designer.hpp

```diff
@@ -93,22 +91,21 @@ class automatic_exhaustive_gate_designer_impl
         const auto all_combinations = determine_all_combinations_of_given_sidbs_in_canvas();
 
         std::vector<Lyt> all_found_gate_layouts = {};
```

Comment:
Are the layouts `found` or `designed`?

Reply:
@marcelwa Both 😅 , but I guess ``designed`` is better. Thanks.